crates/ln-dlc-node/src/node/invoice.rs

```diff
@@ -128,21 +128,33 @@ where
         }
     }
 
-    /// Creates a fake channel id needed to intercept payments to the provided `target_node`
+    /// Creates a fake channel ID needed to intercept payments to the provided `target_node`
     ///
-    /// This is mainly used for instant payments where the receiver does not have a lightning
-    /// channel yet, e.g. Alice does not have a channel with Bob yet but wants to
-    /// receive a LN payment. Clair pays to Bob who opens a channel to Alice and pays her.
+    /// This is mainly used for instant payments where the receiver does not have a Lightning
+    /// channel yet, e.g. Alice does not have a channel with Bob yet but wants to receive an LN
+    /// payment. Claire pays to Bob who opens a channel to Alice and pays her.
     ///
-    /// - `jit_fee_rate_basis_points`
-    /// Fee rate to be charged for opening just in time channels. Rate is in basis points, i.e.
-    /// 100 basis point=1% or 50=0.5%
+    /// # Arguments
+    ///
+    /// - `jit_fee_rate_basis_points`: Fee rate to be charged for opening just in time channels.
+    /// Rate is in basis points, i.e. 100 basis point=1% or 50=0.5%. This argument is optional. If
+    /// not provided, we default to the value configured in [`Node::channel_config`].
     #[autometrics]
     pub fn create_intercept_scid(
         &self,
         target_node: PublicKey,
-        jit_fee_rate_basis_point: u32,
+        jit_fee_rate_basis_point: Option<u32>,
```

Comment:
❓ Do we still need that argument? I might have missed something, but isn't it always set to `None`?

Reply:
Good point, I don't think it's worth keeping it around!